src/components/jnlp-page/utilities/ImageSlider.astro

```diff
@@ -0,0 +1,95 @@
+---
+
+---
+
+<script defer src="https://unpkg.com/alpinejs@3.2.3/dist/cdn.min.js"></script>
```

Comment:
Whoa, Alpine.js!? This is going to need some strong justification.

It looks like all of this could be done without a new dependency and with server-side rendering e.g. using Svelte's transitions. Happy to help if needed.

If we do go with it, please `pnpm install alpinejs` and import it instead of using a CDN.